### ISSUE TITLE
Play/Pause media key on keyboard plays/pauses videos in Caprine

### ISSUE THREAD

**Issue by GeneralKenobi1** [sindresorhus/caprine#1385]
My keyboard has a special play/pause key. I use it to play and pause my music. Some other programs are also responsive to it, such as my media player, but those programs have an option to disable it.

I notice if one of my messages in Caprine has a video, the play/pause button works on it. Even if program is minimized. Even if that message is not the active message. It can be a little strange to have both music and a random video playing at the same time. Or have music playing with one push, then I push it again and the video plays.

Consider adding code to block the media play/pause button from affecting Caprine videos.

Windows 7

![2020-06-22_085832](https://user-images.githubusercontent.com/43397938/85308861-a10eff80-b466-11ea-82eb-b2e901dce65f.png)

**Comment by mquevill** [comment]
I agree with this; there are very few cases where someone would use Caprine as a media player, so I think the default should be to ignore system media keys. If someone wants that functionality, it could be a selectable option, but is not necessary.
